plugins/node/opentelemetry-instrumentation-bunyan/src/types.ts

```diff
@@ -21,5 +21,24 @@ import { InstrumentationConfig } from '@opentelemetry/instrumentation';
 export type LogHookFunction = (span: Span, record: Record<string, any>) => void;
 
 export interface BunyanInstrumentationConfig extends InstrumentationConfig {
+  /**
+   * Whether to disable the automatic sending of log records to the
+   * OpenTelemetry Logs Bridge API.
+   * @default false
+   */
+  disableLogsBridge?: boolean;
```

Comment:
Maybe call this one disableLogs, docs mention something about not using the "bridge" term in the code https://opentelemetry.io/docs/specs/otel/logs/bridge-api/#artifact-naming

Reply:
I'm good with avoiding the "bridge/bridging" terminology.  `disableLogs` is a little ambiguous.  What about `disableLogSending` or `disableLogForwarding`?  Then, for example in the README.md the example could read:

```js
logger.info('hi');
// 1. Log records will be sent to the SDK-registered log record processor, if any.
//    This is called "log sending".
```

The OTel Java instrumentation for the equivalent for the Log4j2 framework describes itself as (https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/log4j/log4j-appender-2.17/library/README.md):

> This module provides a Log4j2 [appender](https://logging.apache.org/log4j/2.x/manual/appenders.html) which forwards Log4j2 log events to the [OpenTelemetry Log SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk/logs).

Note that it uses "appender" in the artifact name (package name `opentelemetry-log4j-appender-2.17`), which was also mentioned in that OTel Logs spec.

Reply:
Yeah both names you propose should be fine, not super picky about that.

Reply:
commit f850a63c for this.

I chose "log sending" over "log forwarding" because "send" is the common verb used at https://opentelemetry.io/docs/specs/otel/logs/#direct-to-collector and elsewhere in the Logs spec.